src/universal/conf/storage.conf

```diff
@@ -0,0 +1,35 @@
+grothendieck {
+
+  # Base directory where all the data used by the node is stored, including blockchain data and private keys
+  # datadir = "/tmp/etc-client/"
+
+  # The unencrypted private key of this node
+  # TODO: rename?
+  # keys-file = ${etc-client.datadir}"nodeId.keys"
```

Comment:
I think better than using substitutions, would be to recognise absolute vs relative paths, and if relative, join with `datadir`.